README.md

```diff
@@ -89,6 +90,9 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Android Studio ile Uygulama Geliştirme Eğitimi](https://www.youtube.com/playlist?list=PL9qDMO9EzLX25NTHm0q7svKLx__OZY8-e)- Levent Yadırga
 - [Android KTX Nedir](https://medium.com/@halilozel1903/android-ktx-nedir-84ecbc615bfb) - [Halil ÖZEL](https://github.com/halilozel1903)
 - [App Inventor Nedir ?](https://medium.com/@halilozel1903/app-inventor-nedir-c37215ae598a) - [Halil ÖZEL](https://github.com/halilozel1903)
+- [Android Programlamada Relative Layout Kullanımı](https://yazilimsanati.net/android-programlamada-relative-layout-kullanimi/) - [Halil ÖZEL](https://github.com/halilozel1903)
+- [Android Programlamada Volley Kullanımı](https://yazilimsanati.net/android-programlamada-volley-kullanimi/) - [Halil ÖZEL](https://github.com/halilozel1903)
+- [Android Nedir ?](https://yazilimsanati.net/android-nedir/) - [Halil ÖZEL](https://github.com/halilozel1903)
```

Comment:
Sadece buna yazıyorum ama alttakilerin hepsi için de geçerli olduğunu düşünebilirsiniz:

Soru işareti (?) öncesinde boşluk bırakılmaz.